Buildings/Controls/OBC/Utilities/PIDWithAutotuning/AutoTuner/AMIGO/PI.mo

```diff
@@ -0,0 +1,70 @@
+within Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.Amigo;
+block PI "Identifies the parameters of a PI controller"
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput kp(min=1E-6)
+    "Connector for the signal of the gain of a first order time-delayed model"
+    annotation (Placement(transformation(extent={{-140,40},{-100,80}}),
+        iconTransformation(extent={{-140,40},{-100,80}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput T(min=0)
+    "Connector for the signal of the time constant of a first order time-delayed model"
+    annotation (Placement(transformation(extent={{-140,-20},{-100,20}}),
+        iconTransformation(extent={{-140,-20},{-100,20}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput L(min=1E-6)
+    "Connector for the signal of the time delay of a first order time-delayed model"
```

Comment:
"Time delay of a first order time-delayed model"

Reply:
ditto